data_management/models/staging/import/sources.yml

```diff
@@ -14,1825 +14,899 @@ sources:
           - name: indic_id
             data_type: text
             description: "Identifiant de l'indicateur"
-            meta:
```

Comment:
put meta property inside a config property as per the documentation https://docs.getdbt.com/reference/resource-configs/meta